pyartifactory/objects.py

```diff
@@ -399,31 +404,107 @@ def delete(self, name: str) -> None:
         """
         self.get(name)
         self._delete(f"api/{self._uri}/{name}")
-        logging.debug("Group %s successfully deleted", name)
+        logger.debug("Group %s successfully deleted", name)
 
 
 class ArtifactoryRepository(ArtifactoryObject):
     """Models an artifactory repository."""
 
     _uri = "repositories"
 
+    # Repositories operations
+    def get_repo(
+        self, repo_name: str
+    ) -> Union[
+        LocalRepositoryResponse, VirtualRepositoryResponse, RemoteRepositoryResponse
+    ]:
```

Comment:
:fire:

Reply:
Thanks, I used "type aliases" everywhere but within the `parse_obj_as` because of the following mypy error:
`error: The type alias to Union is invalid in runtime context`